### DISCUSSION TITLE
Copy paste between two windows

### DISCUSSION THREAD

**Discussion by jtannous** [coder/code-server#4761] [Q&A]
Hi there,

I'm new to Code-Server, and I wanted to start my post by complimenting to this awesome piece of software.

Now to my main issue:
I'm trying to copy paste files\folders between two windows, but it doesn't seem to work, any ideas?

**Comment by jsjoeio** [comment]
I don't think that feature is supported unfortunately. You'll probably have to copy the file via the Integrated Terminal
